discovery_engine_core/providers/Cargo.toml

```diff
@@ -5,21 +5,21 @@ license = "AGPL-3.0-only"
 edition = "2021"
 
 [dependencies]
-chrono = { version = "0.4.19", default_features = false, features = ["clock", "serde"] }
+chrono = { version = "0.4.19", default-features = false, features = ["clock", "serde"] }
 derive_more = { version = "0.99.17", default-features = false, features = ["display"] }
 displaydoc = "0.2.3"
 itertools = "0.10.3"
 lazy_static = "1.4.0"
 maplit = "1.0.2"
-regex = { version = "1.5.5", features = ["unicode-gencat"] }
-reqwest = { version = "0.11.9", features = ["json", "rustls-tls"], default-features = false }
-serde = { version = "1.0.136", features = ["derive"] }
-serde_json = "1.0.79"
+regex = { version = "1.5.6", features = ["unicode-gencat"] }
+reqwest = { version = "0.11.11", default-features = false, features = ["json", "rustls-tls"] }
+serde = { version = "1.0.137", features = ["derive"] }
+serde_json = "1.0.81"
 serde_path_to_error = "0.1.7"
-thiserror = "1.0.30"
+thiserror = "1.0.31"
 url = "2.2.2"
 
 [dev-dependencies]
 claim = "0.5.0"
-tokio = { version = "1.17.0", features = ["rt", "macros"] }
-wiremock = "0.5.10"
+tokio = { version = "1.19.2", default-features = false, features = ["rt", "macros"] }
```

Comment:
:grey_question:  I guess `default-features = false` was previously just left out by mistake? (since in other places it appears)

Reply:
mhh, i double-checked and `tokio` actually doesn't include anything by default. since we don't save space, we also don't have to exclude this explicitely, i'll remove it.